run/README.md

```diff
@@ -1,116 +0,0 @@
-# Google Cloud Run .NET Samples

```

Comment:
Just calling out that this one doesn't have an equivalent in the new location. That might be totally fine, just in case you missed it.

Reply:
I removed it, as they removed it from Node.js samples as well. I don't think it's maintainable to have an uber README.md anyway. Each sample folder has its own README.md and that should be enough.

Reply:
Yes, I agree that an uber readme is not the best.
I'm rerunning Kokoro, there was a failure before that I couldn't quite pinpoint, and will merge on green unless you say otherwise.

Reply:
Thanks @amanda-tarafa